### PR TITLE
App styling

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Added Spinner on,list loading
Milestone 2: Added Style on list
Milestone 2: Added background colors

### CODE REVIEW

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Hector096 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Your PR summary needs to be more detailed briefly listing the changes done in this branch. Kindly make changes in your PR summary. :pray: 
- [ ] You did really great job. I would recommend making a good alignment in the scores list to make it more readable and attractive.
image

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,62 +1,80 @@
...
-const alertBox2 = document.getElementById('alert-box2');
+const listBox = document.getElementById('listbox')
+const form = document.getElementById('form')
+const name = document.getElementById('name')
+const score = document.getElementById('score')
+const refresh = document.getElementById('refresh')
+const alertBox = document.getElementById('alert-box')
+const alertBox2 = document.getElementById('alert-box2')
+const loader = document.getElementById('loader')
+const listBoxDiv = document.getElementById('list-box-div')
```
Comment (cyonii) [inline src/index.js]: - [ ] Your Github workflow action is failing due to missing semicolons in your code, kindly fix this issue.
You can run the code below to install the linter packages in order to get tips or even auto-fixes right in your IDE. 
```shell
# For JavaScript

npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
```

```shell
# For style

npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
```

**Review by cyonii — CHANGES_REQUESTED** [review]
Hi @Hector096, 

Good job so far implementing the requested changes!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review to see the changes you need to make._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**